.github/ISSUE_TEMPLATE/release_tracking.md

```diff
@@ -0,0 +1,77 @@
+---
+name: 🚋 Release cycle tracking
+about: Create a new release cycle tracking issue for a minor release
+title: Tasks for v<release-tag> release cycle
+labels: ''
+assignees: ''
+
+---
+
+Please see the corresponding section in [release-tasks.md](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md) for documentation of individual tasks.  
+
+## Tasks
+
+**Notes**:
+* Weeks are only specified to give some orientation.
+* The following is based on the v1.4 release cycle. Modify according to the tracked release cycle.
+
+Week -3 to 1:
+* [ ] [Release Lead] Set a tentative release date for the minor release
+* [ ] [Release Lead] Assemble release team
+
+Week 1:
+* [ ] [Release Lead] Finalize release schedule and team
+* [ ] [Release Lead] Prepare main branch for development of the new release
+* [ ] [Release Lead] Create a new GitHub milestone
+* [ ] [Communications Manager] Add docs to collect release notes for users and migration notes for provider implementers
+* [ ] [Communications Manager] Update supported versions
+
+Week 1 to 4:
+* [ ] [Release Lead] [Track] Remove previously deprecated code
```

Comment:
I see a lot of task/chores assigned to the release lead. The way I envisioned the role initially was more of a process leader/people organizer:

> responsible for coordinating release activities, assembling the release team, taking ultimate accountability for all release tasks to be completed on time, and ensuring that a retrospective happens. The lead is also responsible for ensuring a successor is selected and trained for future release cycles.

Ultimately the lead is meant to coordinate all activities (eg. create this issue and keep track of tasks, and deal with the people aspect of it (e.g. dealing with a situation where one of the other team members drops out midway), etc. 

As much as possible I think we should avoid assigning direct tasks to them, and if we have a lot of tasks that don't fit in the other buckets maybe we need to rethink the roles / add a role / update their descriptions slightly. The only exception, maybe, is the "Cut vx.y.z release" type tasks.

Reply:
I'm re-reading and maybe that's what you meant by `[Track]`? If that's the case it wasn't super obvious (to me at least) on first read

Reply:
I tried to express this by adding the following note at the top:

> * If a task is prefixed with `[Track]` it means it should be ensured that this task is done, but the folks with 
  the corresponding role are not responsible to do it themselves. 

I'm open to alternatives to express this.

Considering that the Track tasks are only tracked by the release-lead.

Are you fine with the remaining tasks?

I added some of them here because they seemed to not fit anywhere else. I'm also not sure how Shadowing works (e.g. pairing sync (probably not a good idea), cc'ing on PRs, discussions in Slack threads, delegating some of the tasks to shadows?)

Reply:
@ykakarap @oscr @furkatgofurov7 do you all have thoughts here?